### PR TITLE
Add DisactivateUserSkillConsumer

### PR DESCRIPTION
**Pull request by SolovyovDmitry**
Add DisactivateUserSkillConsumer which deletes UserSkills related to user when it gets deleted in UserService

### CODE REVIEW

File: src/SkillService.Broker/Consumers/DisactivateUserSkillConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using LT.DigitalOffice.Models.Broker.Publishing;
+using LT.DigitalOffice.SkillService.Data.Interfaces;
+using MassTransit;
+
+namespace LT.DigitalOffice.SkillService.Broker.Consumers
+{
+  public class DisactivateUserSkillConsumer : IConsumer<IDisactivateUserPublish>
```
Comment (MariyaSin) [inline src/SkillService.Broker/Consumers/DisactivateUserSkillConsumer.cs]: DisactivateUserSkillsConsumer

File: src/SkillService.Broker/Consumers/DisactivateUserSkillConsumer.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      _repository = repository;
+    }
+
+    public async Task Consume(ConsumeContext<IDisactivateUserPublish> context)
+    {
+      await _repository.RemoveAsync(
+        context.Message.UserId, 
+        null);
+    }
```
Comment (MariyaSin) [inline src/SkillService.Broker/Consumers/DisactivateUserSkillConsumer.cs]: update skill cout

File: src/SkillService.Models.Dto/Configuration/RabbitMqConfig.cs [hunk 3]
```diff
@@ -8,5 +9,8 @@ public class RabbitMqConfig : BaseRabbitMqConfig
   {
     [AutoInjectRequest(typeof(IGetUserSkillsRequest))]
     public string GetUserSkillsEndpoint { get; set; }
+
+    [AutoInjectRequest(typeof(IDisactivateUserPublish))]
```
Comment (MariyaSin) [inline src/SkillService.Models.Dto/Configuration/RabbitMqConfig.cs]: do not use autho inject for consumers

File: Common.Configurations.targets [hunk 4]
```diff
@@ -35,7 +35,7 @@
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
     <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.56" />
     <PackageReference Include="LT.DigitalOffice.Kernel.BrokerSupport" Version="1.0.4" />
-    <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.67" />
+    <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.81" />
```
Comment (MariyaSin) [inline Common.Configurations.targets]: pltase update to 82 v

**Review by MariyaSin — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**